### ISSUE TITLE
[docs] Outdated/broken link related to Automated testing

### ISSUE THREAD

**Issue by mnigh** [cds-snc/platform-forms-client#1777]
# Description

The **Accessibility Tracker** markdown is in incorrect. The URL it should be linked to (https://a11y-tools-query.herokuapp.com/revisions/cds-snc%2Fforms) also appears to be broken.

https://github.com/cds-snc/platform-forms-client/blob/4f3c0d59b6d2da2be0038aa478f8a5449984e11e/documentation/Forms/AccessibilityReport.stories.mdx?plain=1#L40

https://github.com/cds-snc/platform-forms-client/blob/4f3c0d59b6d2da2be0038aa478f8a5449984e11e/documentation/Forms/AccessibilityReport.stories.mdx?plain=1#L87

**Comment by srtalbot** [comment]
Thanks you for your github issue @mnigh! 

We not longer use that specific automated testing tool, but [you can read more about our approach to accessibility testing here.](https://digital.canada.ca/2023/02/16/how-were-building-gc-forms-our-4-accessible-approaches/)

We'll remove the link.
